src/emulator/hubExport.ts

```diff
@@ -289,6 +304,25 @@ export class HubExport {
       throw new FirebaseError(`Failed to export storage: ${await res.response.text()}`);
     }
   }
+
+  private async exportDataConnect(metadata: ExportMetadata): Promise<void> {
+    void trackEmulator("emulator_export", {
+      initiated_by: this.options.initiatedBy,
+      emulator_name: Emulators.DATACONNECT,
+    });
+    const dataconnectExportPath = path.join(this.tmpDir, metadata.dataconnect!.path);
+    if (fs.existsSync(dataconnectExportPath)) {
+      fse.removeSync(dataconnectExportPath);
```

Comment:
Shouldn't we do this after checking for running?

Reply:
Good call.